src/WebJobs.Script.WebHost/Resources/AssembliesRemoved.txt

```diff
@@ -0,0 +1,60 @@
+mscordaccore_amd64_amd64_4.6.28207.03
+sos
+sos.netcore
+sos_amd64_amd64_4.6.28207.03
+microsoft.aspnetcore.authentication.facebook
+microsoft.aspnetcore.authentication.google
+microsoft.aspnetcore.authentication.jwtbearer
```

Comment:
@fabiocav - these assemblies were present in 2.2 but missing in 3.1 shared framework. Using this file to log the location of the loaded assembly. This is needed to figure out if the missing assemblies are part of app content or not.